eng/update-band.ps1

```diff
@@ -1,25 +1,41 @@
 [CmdletBinding(PositionalBinding=$false)]
 Param(
-    [Parameter(Mandatory = $true)]
+    [Parameter(Mandatory = $false)]
     [string]$Remote,
 
-    [Parameter(Mandatory = $true)]
-    [Alias("branch-1xx")]
+    [Parameter(Mandatory = $false)]
     [string]$Branch1xx,
 
+    [switch]$Continue = $false,
+
     [switch][Alias('h')]$help
```

Comment:
Sorry I didn't catch this on the first review but there's no need to have a help option for PowerShell. It's integrated into PowerShell itself: `Get-Help update-band.ps1`. To provide descriptions for the parameters, you can use comment-based help: https://learn.microsoft.com/powershell/scripting/developer/help/examples-of-comment-based-help